### PR TITLE
added test cases for evaluation form in mobile view

### PR DESCRIPTION
**Pull request by chadiegilnerubia**
Additional Test cases for evaluation form in mobile view

### CODE REVIEW

File: __tests__/pages/evaluations/[id]/evaluations.spec.ts [hunk 1]
```diff
@@ -202,6 +207,20 @@ test.describe("User - Evaluations", () => {
       await expect(page.getByTestId("OptionButton4")).toBeVisible()
       await expect(page.getByTestId("OptionButton5")).toBeVisible()
       await expect(page.getByTestId("OptionButton6")).toBeVisible()
+
+      const saveButton = page.getByRole("button", { name: "Save", exact: true })
+      await saveButton.waitFor({ state: "visible" })
+      // eslint-disable-next-line @typescript-eslint/strict-boolean-expressions
```
Comment (jatnerubia) [inline __tests__/pages/evaluations/[id]/evaluations.spec.ts:213]: avoid using this

**Review by jatnerubia — CHANGES_REQUESTED** [review]
^^

File: __tests__/pages/evaluations/[id]/evaluations.spec.ts (hunk 1 above)
Comment (chadiegilnerubia) [reply __tests__/pages/evaluations/[id]/evaluations.spec.ts]: done thanks

**Review by emmaacha — APPROVED** [review]
lgtm 🚀

**Review by jatnerubia — APPROVED** [review]
🚀

**Pull request merged**